pkg/duck/v1alpha1/identifiable.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"k8s.io/apimachinery/pkg/runtime"
 	"knative.dev/pkg/apis"
 	"knative.dev/pkg/kmeta"
 
 	duckv1alpha1 "github.com/google/knative-gcp/pkg/apis/duck/v1alpha1"
 )
 
 type Identifiable interface {
+	// runtime.Object can be removed once the old Topic and PullSubscription are removed.
+	runtime.Object
```

Comment:
why is this one needed?

Reply:
The event recorder requires a `runtime.Object`. In the `deleteOldPubSub*CO` methods, I write an event if the old CO is not owned by the pubsubable.